tests/getting-started-tutorial-test.js

```diff
@@ -0,0 +1,75 @@
+module.exports = {
+    tags: ['getting-started-tutorial-test'],
+    'Smoke test getting-started-tutorial' : function (client) {
+      client
+        .url('http://localhost:3000')
+        .pause(1000);
+
+      // Variables for max waiting times
+
+      var apiCallTimeout = 10000;
+      var alertBoxTimeout = 6000;
+
+      // Checking that the page loaded
+
+      client.expect.element('body').to.be.present;
+
+      // Checking that the command logs are showing
+
+      client.waitForElementVisible('span.log-string', apiCallTimeout);
+
+      // Checking that the message input is showing
+
+      client.expect.element('input#order').to.be.present;
+   
+      // Clicking 'Encrypt' w/o having the function added or a message inputted and checking that an alert-box appears
+
+      client.click('button#encrypt-button');
+      client.waitForElementVisible('div#alert-box', alertBoxTimeout);
+      client.assert.containsText('div#alert-box', 'Please enter an order').pause;
+      client.waitForElementNotVisible('div#alert-box', alertBoxTimeout);
+
+      // Entering 'Set phasers to stun', clicking encrypt, and checking that an alert-box appears
+
+      client.setValue('input#order', 'Set phasers to stun');
+      client.click('button#encrypt-button');
+      client.waitForElementVisible('div#alert-box', alertBoxTimeout);
+      client.assert.containsText('div#alert-box', 'WRITE ME: Copy and paste encryptData into app/get-started-by-writing-me.js');
+      client.waitForElementNotVisible('div#alert-box', alertBoxTimeout);
```

Comment:
There's a decent amount of duplication of things here that could pretty easily be solved by using the NightWatch page objects feature like I have setup in the IronWeb repo (apologies for you not having access earlier). You could move this chunk of code out of here to a generic place and then be able to just call `page.clickEncryptButton` and abstract away the actual elements that have to be clicked. That would also allow this to grow better because it moves the majority of your CSS selectors into a single place that can be updated once if the underlying classes/HTML change.